source/_sass/page.scss

```diff
@@ -5,17 +5,17 @@ body {
 }
 .navigation-wrapper {
 	@include container;
-	padding: 2em 0 1em;
+	padding: 40px 0;
 	font-family: $heading-font;
-	font-weight: 700;
-	text-transform: uppercase;
+	height:150px;
+	box-sizing:border-box;
+	border-bottom:1px solid #EBEBEB;
```

Comment:
Color `#EBEBEB` should be written as `#ebebeb`<br>Line should be indented with spaces, not tabs<br>Colon after property should be followed by one space